### PR TITLE
Fix keymap assignments (#442).

### PR DESCRIPTION
**Pull request by pq**
Fixes: #442.

/cc @devoncarew

### CODE REVIEW

File: src/io/flutter/actions/FlutterAppAction.java [hunk 1]
```diff
@@ -17,21 +18,22 @@
...
       getTemplatePresentation().setEnabled(myIsApplicable.compute() && isRunning());
     }
   };
 
-  public FlutterAppAction(ObservatoryConnector connector, String text, String description, Icon icon, Computable<Boolean> isApplicable) {
+  public FlutterAppAction(ObservatoryConnector connector, String text, String description, Icon icon, Computable<Boolean> isApplicable, @NotNull String actionId) {
     super(text, description, icon);
     myConnector = connector;
     myIsApplicable = isApplicable;
+    ActionManager.getInstance().registerAction(actionId, this);
```
Comment (devoncarew) [inline src/io/flutter/actions/FlutterAppAction.java]: This object is registering itself globally in its ctor; I assume this ctor is only called once?

**Comment by devoncarew** [comment]
lgtm! less code and sharing the same action instance

File: src/io/flutter/actions/FlutterAppAction.java (hunk 1 above)
Comment (pq) [inline src/io/flutter/actions/FlutterAppAction.java]: Yep.  It should only get called once (when contributed via plugin registration).  I'll look a little deeper to verify though (and maybe add a guard to be safe).

Comment (pq) [inline src/io/flutter/actions/FlutterAppAction.java]: Guard added.

**Pull request merged**